File: Python/Easy/1518_Water_Bottles/solution.py
class Solution:
    def numWaterBottles(self, numBottles: int, numExchange: int) -> int:
        count = numBottles

        while numBottles >= numExchange:
            rem = numBottles % numExchange
            numBottles = numBottles // numExchange
            count += numBottles
            numBottles += rem
        return count

File: Golang/Easy/1518_Water_Bottles/solution.go
package waterbottles

func numWaterBottles(numBottles int, numExchange int) int {
	count := numBottles

	for numBottles >= numExchange {
		rem := numBottles % numExchange
		numBottles = numBottles / numExchange
		count += numBottles
		numBottles += rem
	}
	return count
}
